backend/.env.example

```
@@ -0,0 +1,25 @@
+REFRESH_TOKEN_EXPIRE_DAYS=7
+
+# Database
+DATABASE_URL=postgresql://user:password@localhost:5432/solfoundry
+
+# Redis
+REDIS_URL=redis://localhost:6379/0
+
+# GitHub OAuth
+GITHUB_CLIENT_ID=your-github-client-id
+GITHUB_CLIENT_SECRET=your-github-client-secret
+GITHUB_CALLBACK_URL=http://localhost:8000/auth/github/callback
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_CALLBACK_URL key should go before the GITHUB_CLIENT_ID key

(UnorderedKey)